Model/Reservation.php

```diff
@@ -608,4 +613,24 @@ public function setTermsAndConditions($termsAndConditions)
 
         return $this;
     }
+
+    /**
+     * @return \string[]
```

Comment:
That `\` is obsolete here. `string` is just a keyword and does not need to be defined as a namespace